plugin/al/usdtransaction/CMakeLists.txt

```diff
@@ -1,17 +1,8 @@
 add_subdirectory(AL/usd/transaction)
 set(LIBRARY_NAME AL_USDTransaction)
 
-set(PY_INIT_FILES
-  ${CMAKE_CURRENT_SOURCE_DIR}/AL/__init__.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/AL/usd/__init__.py
-  ${CMAKE_CURRENT_SOURCE_DIR}/AL/usd/transaction/__init__.py
-)
-
 # copy to build location for module tests
-foreach(INPUT_FILE ${PY_INIT_FILES})
-  string(REPLACE ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR} OUTPUT_FILE ${INPUT_FILE})
-  execute_process(COMMAND "${CMAKE_COMMAND}" -E copy "${INPUT_FILE}" "${OUTPUT_FILE}")
-endforeach()
+file(COPY AL DESTINATION ${CMAKE_CURRENT_BINARY_DIR} FILES_MATCHING PATTERN "*.py")
```

Comment:
Why does this need to be changed? What are you fixing here?

Reply:
It's a clean-up, improving readability.